### PR TITLE
Update translations add uk

### PR DESCRIPTION
**Pull request by demkodo**
errors restored by translator, pushing live to see if the errors have indeed been restored...

### CODE REVIEW

**Comment by joelanders** [comment]
I think GH used to have a graph-like visualization that showed the commits differing between branches, but now on https://github.com/equalitie/deflect-docs/branches, it just says something like "this branch is 3 commits ahead of and 5 commits behind master."

Anyway, that means this branch isn't forked off from the most recent commit in master (it's branched off 5 behind). If some of those 5 commits touch some of the same files you're changing in this branch, git doesn't know which changes to keep. Since we know actually that the files in this branch are newer snapshots than the ones in master, we can tell git to put these changes *after* the changes in master with `git rebase master` (this is assuming you have this branch checked out; the command reads like "git rebase (this onto) master").

Sometimes, there might be rebase conflicts also, but I think they're less common than merge conflicts because with rebase you're telling git which order to apply to changes in (and in a merge, you're telling git to apply all the changes at once). Also, I think these merging/rebasing algorithms are one of the Hard Problems of version control systems, so I don't worry about not understanding it :D

TLDR: `git checkout update-translations-add-uk && git rebase master && git push origin update-translations-add-uk --force` then come back here and it should be mergable.
